config/routes.rb

```diff
@@ -14,6 +14,12 @@
   namespace :my do
     get :dashboard, to: 'dashboard#index'
 
+    resources :events do
```

Comment:
add `only: []` to this, so it won't generate the 7 default actions (which we are not going to use)